sdk/go/pulumi/context_test.go

```diff
@@ -631,6 +631,26 @@ func TestWithValue(t *testing.T) {
 	assert.Equal(t, newCtx.state, testCtx.state)
 }
 
+func TestExportMap(t *testing.T) {
+	t.Parallel()
+
+	var output map[string]Input
+	err := RunErr(func(ctx *Context) error {
+		ctx.Export("first", String("hello"))
+		ctx.Export("second", String("world"))
+
+		output = ctx.GetCurrentExportMap()
```

Comment:
I guess this also works if you call this outside of `RunErr`? That feels a bit nicer and how I'd likely use it in end-user unit test.